o-fish-ios/Views/PreboardingFlow/PreboardingView/PreboardingView.swift

```diff
@@ -175,12 +175,18 @@ struct PreboardingView: View {
             if let realmReports = realmReports {
                 for report in realmReports {
                     if let vessel = report.vessel {
-                        if uniqueIdentifiers.filter({$0 == vessel.permitNumber }).isEmpty && !vessel.name.isEmpty {
-                            uniqueIdentifiers.append(vessel.permitNumber)
+                        if uniqueIdentifiers.filter({$0 == vessel.permitNumber }).isEmpty &&
```

Comment:
hard to understand, can we make it more readable?

Reply:
I tried to refactor it.